packages/dcos-diagnostics/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-diagnostics.git",
-    "ref": "701716319277efad41b760575eaec2683a099b78",
+    "ref": "096c5771987e2f489c9ba0369b080f1bb99bce03",
```

Comment:
Shouldn't this be f5abafecd68821bd7dc0b9d09766d4d72f99667e to also include the `err` shadowing fix?

Reply:
It can go in next bump. I'd like to have the same version in 1.12 and 1.13